ts/features/itwallet/common/utils/itwClaimsUtils.ts

```diff
@@ -42,7 +42,11 @@ export enum WellKnownClaim {
    * Claim used to display a QR Code for the Disability Card. It must be excluded from the common claims list
    * and rendered using a {@link QRCodeImage}
    */
-  link_qr_code = "link_qr_code"
+  link_qr_code = "link_qr_code",
+  /**
+   * Claim used to display the attachments of a credential
```

Comment:
Should we specify which credential has this well known claim for clarity's sake?

Reply:
Addressed in https://github.com/pagopa/io-app/pull/6154/commits/448d3dc59d6ed2d929e46e262c526d2655d82a07